frontend/express/views/login.html

```diff
@@ -27,82 +28,139 @@
-                    <a id="forgot-password" href="./forgot"  data-localize="login.forgot"></a>
-                </div>
-            </form>
+		<div class="footer">
+			<div>
+				<p style="margin: 0px;">© 2021 Countly Ltd.</p>
+				<div style="margin: 0px; color: #0166D6;">
+					<a href="https://count.ly/">Countly</a>
+					<a href="https://count.ly/legal/terms-of-service">Terms</a>
+					<a href="https://count.ly/legal/privacy-policy">Privacy</a>
+					<a href="https://resources.count.ly/">Documentation</a>
+					<a href="https://support.count.ly/hc/">Help Center</a>
+				</div>
+			</div>
 		</div>
-        <% if (typeof inject_template.html != "undefined") { %>
-            <%- inject_template.html %>
-        <% } %>
 	</div>
 	<script language="javascript" type="text/javascript" src="javascripts/dom/jquery/jquery.js"></script>
 	<script language="javascript" type="text/javascript" src="javascripts/utils/prefixfree.min.js"></script>
 	<script language="javascript" type="text/javascript" src="javascripts/utils/store+json2.min.js"></script>
 	<script language="javascript" type="text/javascript" src="javascripts/utils/jquery.i18n.properties-min-1.0.9.js"></script>
 	<script language="javascript" type="text/javascript" src="javascripts/utils/jquery.xss.js"></script>
 	<script language="javascript" type="text/javascript" src="javascripts/pre-login.js"></script>
+	<script language="javascript" type="text/javascript" src="https://cdn.jsdelivr.net/jquery.validation/1.16.0/jquery.validate.min.js"></script>
+	<script language="javascript" type="text/javascript" src="https://cdn.jsdelivr.net/jquery.validation/1.16.0/additional-methods.min.js"></script>
 	<script>
         var countlyTitle = "<%- countlyTitle %>";
 		$(document).ready(function() {
-			<% if (message && message.length > 0){ %>
+			$('.cly-vue-notification__alert-box').show();
+			<% if (message && message.length >0 ){ %>
 				showMessage("<%= message %>");
+			<% } 
+			else{ %>
+				$(".cly-vue-notification__alert-box").css('display','none');
 			<% } %>
             if (jQuery.i18n.map["login.title"]) {
                 document.title = countlyTitle + " | " + jQuery.i18n.map["login.title"];
             }
 		});
-		
 		$(document).bind('clyLangChange', function() {
             if (jQuery.i18n.map["login.title"]) {
                 document.title = countlyTitle + " | " + jQuery.i18n.map["login.title"];
             }
 		});
+		$(document).ready(function() {
+			$("*").click(function(event){
+				if ($(".top-button").hasClass("active") && !$(event.target).is(".top-button") && !$(event.target).is("#active-lang")) {
+					$('.top-button').removeClass('active');
+				}
+			});
+		});
+		$(document).ready(function() {
+			$("*").click(function(event){
+				if ($(event.target).is(".top-button") || $(event.target).is("#active-lang")) {
+					$('#active-lang').css('border','1px solid #81868D');
+				}
+				else{
+					$('#active-lang').css('border','');
+				}
+			});
+		});
```

Comment:
What do these lines do? :)

Reply:
clicking on the languages button opens the menu, clicking on the button again closes the menu. in the first piece of code; I aimed to close it when it is clicked off-target. in the second piece of code; if the menu is open, the border should be in a different color and in a single color, when it closes, it should return to the hover state and the css should be removed I intended this.

Reply:
clicking on the languages button opens the menu, clicking on the button again closes the menu. in the first piece of code; I aimed to close it when it is clicked off-target. in the second piece of code; if the menu is open, the border should be in a different color and in a single color, when it closes, it should return to the hover state and the css should be removed I intended this.